iFixit/react-commerce
# Header 20th logo

:tada: Happy 🥳 20th 🎂 iFixit! 

## QA
Please check that the header now uses the appropriately sized logo from [here](https://github.com/iFixit/ifixit/issues/47658#issue-1692997866). The logo is a tad larger than the current one, but that is to account for the extra things in the design. The actual iFixit circle logo and 'IFIXIT' parts should still be the same size.

Please also check the hamburger side menu on small screens because we show the logo here.

Closes iFixit/ifixit#47658
